Finjector.Web/ClientApp/tsconfig.json

```diff
@@ -20,7 +16,5 @@
     "noEmit": true,
     "jsx": "react-jsx"
   },
```

Comment:
think about adding `    "types": ["vitest/globals", "@testing-library/jest-dom"],` which will allow you to use types natively in testing w/o the awkward prefixes